impl/src/test/java/com/sun/faces/context/ExternalContextImplWebsocketTest.java

```diff
@@ -0,0 +1,75 @@
+package com.sun.faces.context;
```

Comment:
Looks good, but needs a license header (you can copy the EPL header from another file and then add your own name)

Reply:
@arjantijms, done, added the license header